tests/cli.rs

```diff
@@ -150,8 +144,18 @@ async fn server_started_twice_fails() {
 
     let second_server = start_server(server_config);
 
-    let exit = second_server.handle.wait();
-    assert_eq!(exit.err().map(|err| err.kind()), Some(io::ErrorKind::Other))
+    let timeout = Instant::now() + Duration::from_secs(1);
```

Comment:
General remark - didn't go in the code to verify. Whenever I do timeout verifications now (not in cothority :(, I try to abstract the time away, so in the test I don't have to wait for 1 second. You can do this either by giving the time as a parameter to the method you're calling, or by passing it a modifiable clock-object.

Reply:
Moving to a `const TIMEOUT`

Reply:
If you test like this I think you don't have another possibility than to actually wait. My proposition is to create a unit test in a way that you can call the module for example with your 'own' clock -> don't read the clock in your module, but let the caller provide the clock.
Or provide a clock object when creating the module, in such a way that you can pass it your own clock module for testing, and then simulate a timeout without to have actually to wait for it.

Reply:
Face2Face: the timeout here is for the settling of the network, not to check if the system timeout correctly